### PR TITLE
[CI] remove useless git am action when build with cxx11 abi (#92)

### PR DESCRIPTION
**Pull request by Orion34C**

### CODE REVIEW

File: scripts/python/tao_build.py [hunk 1]
```diff
@@ -471,9 +471,6 @@ def bazel_build(target, flag=""):
             execute("sed -i \
                 's/values\[i\] = internal::InnerMostDimReducer<Self, Op>::reduce(\*this, firstIndex + i \* num_values_to_reduce,$/CoeffReturnType t = internal::InnerMostDimReducer<Self, Op>::reduce(\*this, firstIndex + i \* num_values_to_reduce, num_values_to_reduce, reducer)\;values\[i\] = t\;/g' \
                 'bazel-tf_community/external/eigen_archive/unsupported/Eigen/CXX11/src/Tensor/TensorReduction.h'")
-        if args.ral_cxx11_abi:
-            # A workaround for build compiler main with gcc 730 when enable cxx11 abi
-            execute("git am ../platform_alibaba/ci_build/patch/workaround-to-fix-cxx11-build.patch")
```
Comment (linearhit) [inline scripts/python/tao_build.py:476]: Do we need to remove the patch file from the repo as well?

**Review by linearhit — CHANGES_REQUESTED** [review]

File: scripts/python/tao_build.py (hunk 1 above)
Comment (Orion34C) [reply scripts/python/tao_build.py:476]: The patch is not in this repo, this is what I want to fix at first, then I found it is not needed for cxx11 abi build, so just need to remove this logic from tao_build.py.

**Review by Yancey1989 — APPROVED** [review]
LGTM

**Review by linearhit — APPROVED** [review]
LGTM

**Pull request merged**